core/kits/assets/js/manager.js

```diff
@@ -3,6 +3,8 @@ import panelView from './panel';
 import panelMenuView from './panel-menu';
 import PanelHeaderBehavior from './panel-header-behavior';
 import Repeater from './repeater';
+import GlobalControlSelect from './globals/global-select-behavior';
+import ControlsCSSParser from '../../../../assets/dev/js/editor/utils/controls-css-parser';
```

Comment:
```suggestion
import ControlsCSSParser from 'elementor/editor/utils/controls-css-parser';
```

Reply:
Uncaught Error: Cannot find module 'elementor/editor/utils/controls-css-parser' @matipojo

Reply:
See the updated suggetion.